zeebe-job-worker/build.gradle.kts

```diff
@@ -0,0 +1,27 @@
+plugins {
+    kotlin("jvm") version "1.8.21"
+    kotlin("plugin.serialization") version "1.4.21"
+    application
+}
+
+group = "org.example"
+version = "1.0-SNAPSHOT"
+
```

Comment:
You should add instruction to use JVM 17:
`java.sourceCompatibility = JavaVersion.VERSION_17`